driver_darwin.go

```diff
@@ -127,7 +130,10 @@ func newDriver(sampleRate, channelNum, bitDepthInBytes, bufferSizeInBytes int) (
 		nbuf = 2
 	}
 
+	ctx, cancel := context.WithCancel(context.TODO())
```

Comment:
`context.Background` should be fine instead of `TODO`.

Reply:
https://golang.org/pkg/context/#Background

they have exactly the same behavior. The difference is Background is suggested to used by the main function, initialization, and tests, and as the top-level Context for incoming requests. And when it is possible to accept a context as a parameter, TODO may used.

Actually context tree is pretty useful in the nested environment.

However, this is not a big deal. Let me change it..

Reply:
Updated

Reply:
Ah I see. I don't plan to change the API to accept the context, but that's good to know.